src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-create-initial/sw-order-create-initial.spec.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @package checkout
+ */
+
+import { mount } from '@vue/test-utils';
+import orderStore from 'src/module/sw-order/state/order.store';
+
+const existingCustomerId = '1234';
+const notExistingCustomerId = '9876';
```

Comment:
This isn't nessecarry, just hard code the ids

Reply:
Ok, I have changed it again.